quipucords/scanner/network/connect.py

```diff
@@ -181,6 +187,7 @@ def run_with_result_store(self, result_store):
         # Clear cache as results changed
         self.result = None
 
+        self.scan_task.end_time = datetime.datetime.now(self.timezone).time()
```

Comment:
I'd move this logic to here:
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scantasks/model.py#L90
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scantasks/model.py#L114
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scantasks/model.py#L124
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scantasks/model.py#L108

Same for scan job:
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scanjob/model.py#L160
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scanjob/model.py#L224
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scanjob/model.py#L252
https://github.com/quipucords/quipucords/blob/master/quipucords/api/scanjob/model.py#L267